iam/api/QuickStartV2/QuickStart.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Google.Apis.Cloudresourcemanager.v1" Version="1.43.0.1859" />
+    <PackageReference Include="Google.Apis.Iam.v1" Version="1.43.0.1835" />
```

Comment:
I just realized about this, sorry. We shouldn't be writing samples for [Google.Apis.Iam.V1](https://www.nuget.org/packages/Google.Apis.Iam.v1/) but for [Google.Cloud.Iam.V1](https://www.nuget.org/packages/Google.Cloud.Iam.v1/). The latter is the recommended client library for the IAM service. The overall sample will be similar but you will have to change the code some to call the recommended client library.
Again, sorry for not catching this earlier. Let me know if I can do anything to help.

Reply:
Scratch this, I just sent an internal email as to why. The short explanation is:
Google.Cloud.Iam.V1 is to be used in conjunction with other Google.Cloud.* libraries. When doing "standalone" IAM operations, the correct library to use is Google.Apis.Iam.V1.
Sorry for the confusion.

Reply:
Sounds good!